draft-yoshino-wish.txt

```diff
@@ -252,11 +252,12 @@ Internet-DraWiSH: A General Purpose Message Framing over Byt  March 2017
    underlying protocols.
 
    The permessage-deflate extension [RFC7692] is defined for the
-   WebSocket protocol, to add a compression mechanism to it.  No
-   extension mechanism is defined for WiSH.  Compression can be
-   implemented by underlying protocols or in the application layer if
-   needed.  What contents are exchanged and in what encoding they are
-   exchanged over WiSH are to be defined by the application layer.
+   WebSocket protocol, to add a compression mechanism to it.  The
+   permessage-deflate extension can be applied to WiSH.  The details are
+   to be specified [BidiwebCompression].
```

Comment:
> I don't think we should link to any github issues from I-D.

Fixed by https://github.com/bidiweb/wish/commit/8640a16ad9cf02cdd2ff5d513293bb08dccafe3e

> p.s. when "WiSH" is mentioned in the I-D, we should qualify it with specific meaning, e.g. WiSH framing etc What do you think?

I've replaced all occurrences of `WiSH framing` with `WiSH` as said in https://github.com/bidiweb/wish/pull/17#issuecomment-299841268 considering WiSH basically means the framing. Do you want to use WiSH as a suffix for building various terms or also make it work alone meaning the framing?

At least, the title of the main I-D sounds like WiSH is the name of the framing. So, we should align with it or change the title to WiSH framing.